companies/vbb.json

```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "vbb",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "travel"
+    ],
+    "name": "VBB Verkehrsverbund Berlin-Brandenburg GmbH",
+    "address": "VBB Verkehrsverbund Berlin-Brandenburg GmbH\nDatenschutz\nHardenbergplatz 2\n10623 Berlin\nDeutschland",
+    "email": "datenschutz@vbb.de",
+    "web": "https://www.vbb.de/",
+    "sources": [
+        "https://www.vbb.de/datenschutz"
+    ],
+    "comments": [
+        "Fragen oder Anregungen zur Datenverarbeitung beim VBB können auch per E-Mail an info@vbb.de oder telefonisch an +49 30 254140 gestellt werden."
```

Comment:
You can move the telephone number into the `"phone"` field

Reply:
The problem in this (as I understood it) is that the phone number is not directly for personal data protection / GDPR request matters.

Reply:
That's fine, if there is no phone number specific to data protection matters, we just fallback to a generic one.